code/iaas/logic/src/main/java/io/cattle/platform/process/containerevent/ContainerEventCreate.java

```diff
@@ -197,6 +209,32 @@ void setNetwork(Map<String, Object> inspect, Instance instance) {
         }
     }
 
+    boolean isIpInNetwork(String ipAddress, Network network) {
+        for (Subnet subnet : objectManager.children(network, Subnet.class)) {
+            if (subnet.getRemoved() == null) {
+                String startIp = NetUtils.getDefaultStartAddress(subnet.getNetworkAddress(), subnet.getCidrSize());
+                long start = NetUtils.ip2Long(startIp);
+                String endIp = NetUtils.getDefaultEndAddress(subnet.getNetworkAddress(), subnet.getCidrSize());
+                long end = NetUtils.ip2Long(endIp);
+                long ip = NetUtils.ip2Long(ipAddress);
+                if (start <= ip && ip <= end)
```

Comment:
@ibuildthecloud The rebase hid our discussion so summarizing:
`getDefaultStartAddress` and `getDefaultEndAddress` actually already compensate for reserved addresses. The start adds 2 and the end subtracts 6, so you end up with a range of:
x.x.x.2 - x.x.x.250.
So are you good with it be inclusive with that range?